2_segmenting_data/README.md

```diff
@@ -0,0 +1,104 @@
+# Segment Schwann Cells from NF1 Data
+
+In this module, I present the pipeline for segmenting nuclei and cytoplasm from the NF1 pilot data.
+
+## Segmentation
+
+The method used for segmentation for both the nuclei and cytoplasm is an algorithm called [CellPose](https://doi.org/10.1038/s41592-020-01018-x). Through the Python implementation of this method, this pipeline can be reproducible and is easy to manipulate for various datasets.
+
+## Nuclei Segmentation
+
+After experimenting with the various parameters in CellPose on all of the pilot NF1 DAPI (nuclei) channel images, I settled on the following parameters for CellPose nuclei segmentation:
+- `model_type : "cyto"` This parameter sets the model for CellPose to use as the cytoplasm model, which I found segments nuclei better than the established nucleus model. 
+More information about the various CellPose models can be found at https://cellpose.readthedocs.io/en/latest/models.html.
+- `channels : [0,0]` This parameter sets the channels to be used by the model to segment, which in the case of nuclei, we do not ned to set channels because the images are greyscale.
+- `diameter : 50` This parameter sets the cell diameter for cells within the image to an extact value. 
+This can also be calculated using the `calculate` button in the CellPose GUI. 
+I found that the segmentation worked better when the diameter was established with a set value. 
+- `flow_threshold : 0` This paramenter decreases the maximum allowed error of the flows for each mask (default is `flow_threshold : 0.4`).
+- `remove_edge_masks : True` This parameter removes any masks from CellPose that are touching an edge of the image.
+
+To show how I determined the best parameters for nuclei segmentation, below is a figure showing two rows of nuclei, with an image of the nuclei without segmentation (left), a segmented nuclei using non-optimal parameters (middle), and a properly segmented nuclei (right). 
+
+![Nuclei_Segmented](Example_Figures/Nuclei_Segmentation_Fig.png)
+
+For **nucleus A**, you can see that it is not completely circular and has an prominent indent. 
+In the middle (non-optimal) image, these parameters cut off parts of the nucleus from being apart of the cell. 
+In the right image, the parameters I used segment this nucleus where it includes all of the cell and keeps its shape. 
+
+For **nucleus B**, it has a smooth and continuous texture with no dark lines indicating two seperate nuclei. 
+In the middle (non-optimal) image, these parameters seperate this one nucleus into two nuclei, which is obviously wrong based on how the nuclei looks. In the right image, the parameters I used properly segmented the nucleus as one nucleus.
+
+## Cytoplasm Segmentation
+
+After experimenting with the various parameters in CellPose on all of the pilot NF1 DAPI (RFP) channel images, we settled on the following parameters for CellPose cytoplasm segmentation:
```

Comment:
I am confused about what this means: `NF1 DAPI (RFP) channel images`

Why is RFP in parentheses?

Reply:
No idea... must have been an accident! I will fix it!